identity/verifier.go

```diff
@@ -2,26 +2,52 @@ package identity
 
 import (
 	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/syndtr/goleveldb/leveldb/errors"
 )
 
 type Verifier interface {
 	Verify(message []byte, signature Signature) bool
 }
 
-type ethereumVerifier struct {
-	peerIdentity Identity
+func NewVerifyIsAuthorized() *verifyIsAuthorized {
+	return &verifyIsAuthorized{}
+}
+
+func NewVerifyIdentity(peerIdentity Identity) *verifyIdentity {
+	return &verifyIdentity{peerIdentity}
 }
 
-func NewVerifier(peerIdentity Identity) *ethereumVerifier {
-	return &ethereumVerifier{peerIdentity}
+type verifyIsAuthorized struct{}
```

Comment:
confusing naming for struct with no fields.
`VerifyIsAuthorized` doesn't make much sense to me.

Reply:
`AuthorizationVerifier`?

Reply:
Renamed